scripts/dts/extract_dts_includes.py

```diff
@@ -494,6 +494,12 @@ def extract_property(node_compat, yaml, node_address, y_key, y_val, names,
     if y_key == 'reg':
         extract_reg_prop(node_address, names, defs, def_label,
                          1, y_val.get('label', None))
+        bus = y_val.get('bus', None)
+        if bus:
+            extract_single(node_address, yaml,
```

Comment:
This is essentially why I dislike this solution.
We end up with lot of ad hoc triage in this function with use of specific extract_<new property>, while I hope we should be able to reduce it thanks to generic scheme that should be provided by yaml description.